policies/templates/gcp_sql_public_ip_v1.yaml

```diff
@@ -29,7 +29,7 @@ spec:
     validation.gcp.forsetisecurity.org:
       rego: | #INLINE("validator/sql_public_ip.rego")
          #
-         # Copyright 2019 Google LLC
+         # Copyright 2018 Google LLC
```

Comment:
Not sure why the year is changed in this file from re-formatting.

Reply:
The template didn't match the Rego.